src/xdocs/checks/misc/descendanttoken.xml

```diff
@@ -36,10 +36,7 @@
             <tr>
               <td>limitedTokens</td>
               <td>Specify set of tokens with limited occurrences as descendants.</td>
-              <td>
-                subset of tokens
-                <a href="../../apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html">TokenTypes</a>
-              </td>
+              <td><a href="../../property_types.html#subset of tokens TokenTypes">subset of tokens TokenTypes</a></td>
```

Comment:
trying to see if this new changes after macros migration be restored to original